src/reducers/modal.js

```diff
@@ -22,6 +22,7 @@ const reducer = handleActions({
 }, {
   isOpen: false,
   currentModal: undefined,
+  transactions: [],
```

Comment:
Can we turn this into a more generic "payload" attribute? I think we'll have more of these parameters we have to give to our modals. What do you think?